lib/master.js

```diff
@@ -195,9 +195,35 @@ Master.prototype._startWorkers = function(remainingWorkers) {
                 type: 'config',
                 body: yaml.dump(self.config)
             });
+
+            var workerExit = function(code) {
```

Comment:
Yup. I'd be :+1: on doing this. Conceptually, worker restarts per se shouldn't happen before the completion of the start-up sequence.